features/step_definitions/kata.rb

```diff
@@ -1,229 +1,103 @@
+  raise "Kata runtime did not install into all worker nodes, only #{installed_node_count} reached, expecting #{expected_node_count}" unless matched[:success]
+  logger.info("#{installed_node_count} out of #{expected_node_count} worker nodes installed with kata runtime")
 end
 
-Given /^I remove kata operator from the#{OPT_QUOTED} namespace$/ do | kata_ns |
-  ensure_admin_tagged
-  step %Q/I store master major version in the clipboard/
-  kata_ns ||= "openshift-sandboxed-containers-operator"
-  step %Q/I switch to cluster admin pseudo user/
-  # 1. remove kataconfig first
-  project(kata_ns)
-  # prereq is that there are no outstanding pods in the cluster with
-  # kata-runtime.  Do the destructive action of removing them so kataconfig can be removed.
-  step %Q/I find all pods running with kata as runtime in the cluster and store them to the :kata_pods clipboard/
-  if cb.kata_pods.count > 0
-    logger.info("All existing pods using kata must be removed before kataconfig can be uninstalled.")
-    logger.info("Removing all pods using kata...")
-    step %Q/I remove all kata pods in the cluster stored in the clipboard/
-  end
-
-  kataconfig_name = BushSlicer::KataConfig.list(user: admin).first.name
-  step %Q/I ensure "#{kataconfig_name}" kata_config is deleted within 1500 seconds/
-  # 2. remove namespace
-  step %Q/I ensure "#{kata_ns}" project is deleted/
+Given /^Valid cluster type for kata tests exists$/ do
+  accepted_platforms = ['gcp', 'azure']
+  iaas_type = env.iaas[:type] rescue nil
+  raise "Kata installation only supports #{accepted_platforms} platforms, #{iaas_type} is not a valid cluster type" unless accepted_platforms.include? iaas_type
+  logger.info("Cluster type #{iaas_type.to_s.upcase} is a valid cluster type")
 end
 
-# assumption is that kata is already installed
-And /^I verify kata container runtime is installed into a worker node$/ do
-  # create a project and install sample app has
-  org_user = user
-  step %Q/I switch to the first user/
-  step %Q/I create a new project/
-  cb.test_project_name = project.name
-  file_path = "kata/example-fedora-kata.yaml"
-  step %Q(I run oc create over ERB test file: #{file_path})
-  raise "Example kata pod creation failed" unless @result[:success]
-  logger.info("Waiting for 'example-fedora-kata' pod to be RUNNING")
-  step %Q/a pod becomes ready with labels:/, table(%{
-    | app=example-fedora-kata-app |
-  })
-  logger.info("Checking for runtime engine match...")
-  # 1. check pod's spec to make sure the runtimeClassName is 'kata'
-  pod_runtime_class_name = pod('example-fedora-kata').runtime_class_name
-  if pod_runtime_class_name != 'kata'
-    raise "Pod's runtimeclass name #{pod_runtime_class_name} should be `kata`"
-  end
-
-  logger.info("Checking for running process `qemu` with the pod...")
-  # 2. check there's a process with the pod name `qemu`
+Given /^Catalog source #{QUOTED} exists in #{QUOTED} namespace$/ do |catalog_source_name, project_name|
```

Comment:
@valtair1891 replace `Catalog` with `catalog`.  Also, isn't it `catalogsource` instead of two words?

Reply:
done